### DISCUSSION TITLE
Add feature: move broken videos to specific folder ?

### DISCUSSION THREAD

**Discussion by rleaver152** [theophanemayaud/video-simili-duplicate-cleaner#32] [Ideas]
Wonderful work - bought on App Store.       Can you add feature to move bad videos to alternate directory?  Seems a quick win since you already test them?

**Comment by theophanemayaud** [comment]
Thanks a ton !!!
Yeah it'd be a good thing to implement. I guess simply implementing a setting to change where to move files upon "removal" : either trash, or some specific folder. Would that work for you ?

You can however do this right now, albeit manually : move them manually out of the trash to the folder you want, after the app has trashed them.

**Reply by theophanemayaud** [reply]
I found a way to do just this very easily : the next update should have a top app bar menu option under tools where you can select another folder than the trash, into which videos should be moved. Does that solve it for you ? 😊

**Reply by theophanemayaud** [reply]
Just pushed the update to the App Store, it should be released after Apple accepts it, in a few hours / days.

**Comment by rleaver152** [comment]
sorry for late reply - thats brilliant.  Yeah any folder that can isolate the bad ones would save masses of time

thanks again

**Reply by theophanemayaud** [reply]
Ohoho i just had a realization ! I think I completely mis-understood what you were saying !!! 
i thought you meant being able to move the duplicate videos to a specific folder, instead of the trash. That’s what I implemented !

but now I think what you meant is : isolate the videos that the app fails to scan ! The problem with moving them is that they might have errors for multiple reasons. For example, if the videos are of size 0, or if the screenshots are all black, or if for some reason the app has a problem (which might not be because of a broken video, just that the app doesn’t support the format or something. 

How would you think it should be done ?
If the app moves them to a specific folder, you will still have to sort through them manually…
Isn’t it enough that they are all listed with the error message in the main panel ?

**Comment by rleaver152** [comment]
Hi 

If the app can manually move them to an ‘errdir’   for later examination,  that would be brilliant.     I already have something similar written in Octave that does this for bad jpgs.    Really good as it fillets out dodgy media which otherwise would break cataloging and other routines I use.

Just listing them doesn’t really help as there is still the manual effort to translate that into an action on the part of the user.


best

Richard

**Reply by theophanemayaud** [reply]
All right !!! Now I get it😅😅😅 

Ok so maybe move only the files that seem to really have problems, like zero file size, or other failure that’s definitely because of a broken file, but not the files that are just black, or other non file related errors ?

**Comment by rleaver152** [comment]
Sure - any file that looks like it has a fault.    The one I use for images has two directories errdir and baddir to distinguish between no-hopers and not sures.

Richard

**Comment by theophanemayaud** [comment]
I'll be working on this in #46 asap so closing this discussion to be continued in the issue directly !

**Comment by theophanemayaud** [comment]
Hey ! Finally got to release this feature via v1.13.4 which should be out in a couple days on macOS !

**Reply by theophanemayaud** [reply]
And now now included in the Windows update as well 👌
